server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -713,7 +713,14 @@ export class SummaryWriter implements ISummaryWriter {
 		let logTail: ISequencedDocumentMessage[] = [];
 
 		if (this.getDeltasViaAlfred) {
-			logTail = await this.deltaService.getDeltas("", this.tenantId, this.documentId, gt, lt);
+			logTail = await this.deltaService.getDeltas(
+				"",
+				this.tenantId,
+				this.documentId,
+				gt,
+				lt,
```

Comment:
I think we need to break the range of `[gt, lt]` to a chunk of 2K max, and send requests one by one/ in parallel. request more than 2K is slow and crashes objectstore if range is too big.

Reply:
The break into chunk of 2K here is not for accommodate the breaking change, but more like performance improvement.

Reply:
yes it is a performance improvement, but I wanted to keep that in a separate PR to keep them simple and short. I can copy over those changes here if you feel strongly about it.

Reply:
Sorry I should be more deligent. I don't know your split changes are ready. 

It up to you. I prefer to merge that perf improvement ASAP. You can use 2nd/3rd pr to do that, or just in this PR, up to you as long as it is ready

Reply:
no worries, will send a PR for that soon, thank you :)